src/controller.coffee

```diff
@@ -60,7 +60,7 @@ class Controller extends EventEnabled
   ###
   reset: (query = null, params = {}, items = []) ->
     @query = query
-    @params = merge {}, @defaults, params, page: 1
+    @params = merge {}, @defaults, params
```

Comment:
If you want to be able to reset page too, just put it in the first parameter of `merge`, like:

```coffeescript
@params = merge page: 1, @defaults, params
```

Reply:
But the change is that "page: 1" is removed. Page not have to be reset.

Reply:
- `page` MUST be `1` if you don't specify another value.
- `reset()` was forcing `page` to always be `1` so the change must ensure the previous requirement is complied, but allow another value to be set if desired. That's why your code is wrong for me.

The `serializeStatus()` method was removing `page` so you're now allowing passing a white list of parameters that shouldn't be removed. OK, then, when de-serializing status you will pass your `page` parameter to the `reset()` method and it should accept it.

But you can't break the default behavior or all layers will stop working properly.

Reply:
done